OrbitGl/ApplicationOptions.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
Since when do we have a header like this? Where does this come from?

Reply:
http://b/148520406

Reply:
done as suggested in the bug.